wiki/osu!_wiki/Contribution_guide/en.md

```diff
@@ -1,106 +1,64 @@
 
-   ![](img/fork-ahead.jpg "OK \(these are your commits\)")
+1. In **your fork** of the osu! wiki, find the dropdown which says `master↓`, and select the branch with your changes.
+2. Click the `Contribute↓` link and press the `Open pull request` button.
+3. Fill in the details according to [Best practices § Opening a pull request](/wiki/osu!_wiki/Contribution_guide/Best_practices#opening-a-pull-request) and click `Create pull request`.
 
-   ![](img/fork-behind.jpg "Potentially bad \(your branch is outdated\)")
+## Review
 
-   ![](img/fork-ahead-behind.jpg "Potentially bad \(your branch is outdated with your commits\)")
+All changes to the osu! wiki are moderated. During that stage, other collaborators point out possible mistakes and ways to fix them, either in free form, or by suggesting direct edits to the pull request. Just like with any other peer review, you need to address the comments by applying the suggestions, or by explaining why you'd rather keep things your way.
 
-3. This isn't much of a problem; that is, if you aren't going to edit a file that was already changed which may be unlikely the more commits you are behind.
+If no one has checked in after a couple of days, try the following:
 
-4. To resolve this, see the [My branch is out of date!](/wiki/osu!_wiki/Contribution_guide/Common_Issues#my-branch-is-out-of-date!) section in the Common Issues article.
+- Make sure you have resolved all suggestions—people may be waiting on your responses.
+- Ask other osu! wiki editors in the [osu!dev Discord server](/wiki/osu!dev_Discord_server) (`#osu-wiki` channel) or in the GitHub comments.
+- To get help with the translation, check a couple of merged pull requests for your language and get in touch with their reviewers and/or authors ([example Github query](https://github.com/ppy/osu-wiki/pulls?q=is%3Apr+is%3Amerged+%5BID%5D)).
+- Ask your friends for a quick look!
 
-5. When you complete step 4, continue to [Editing online or locally](#editing-online-or-locally).
+In addition to a manual review process, the osu! wiki repository has [a set of automated checks](/wiki/osu!_wiki/Maintenance#ci-checks), which ensure that your changes follow the common style of the wiki and are free from formatting errors. To see their status, open the `Actions` tab of your pull request and follow the diagnostic messages under the errors, if there are any.
 
-## Editing online or locally
+## Merge
 
-At this point, you have two choices:
+One of the [wiki maintainers](/wiki/osu!_wiki/Maintenance/List_of_maintainers) will check your changes after they are reviewed and approved by other people. If nothing happens after a reasonable amount of time, ask in the comments of the pull request, or in the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server). Your changes will appear on the osu! wiki shortly after the merge (in rare cases, this may take up to five hours).
```

Comment:
```suggestion
One of the [wiki maintainers](/wiki/osu!_wiki/Maintenance/List_of_maintainers) will check your changes after some time. If nothing happens after a reasonable amount of time, ask in the comments of the pull request, or in the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server). Your changes will appear on the osu! wiki shortly after the merge (in rare cases, this may take up to five hours).
```
or something better worded, as its kinda not true that a review is a condition for a maintainer review